gmm/handler.go

```diff
@@ -1983,6 +1983,12 @@ func HandleServiceRequest(ue *context.AmfUe, anType models.AccessType,
 			ue.ConfigurationUpdateMessage = nil
 		}
 	case nasMessage.ServiceTypeData:
+		var plmnAccept bool
+		plmnAccept = context.InTacList(ue.Tai, ue.RanUe[anType].Tai)
+		if !plmnAccept {
+			gmm_message.SendServiceReject(ue.RanUe[anType], nil, nasMessage.Cause5GMMPLMNNotAllowed)
```

Comment:
`plmnAccept` will be false if the PLMN is correct but the TAC is not. In that case, we should reject with `Cause5GMMTrackingAreaNotAllowed`.